examples/kubernetes/README.md

```diff
@@ -0,0 +1,20 @@
+Kubernetes Example
+===
+
```

Comment:
It might be nice to have something here for getting k8s set up quickly locally (minikube?)

Reply:
Sure sounds good let me add it.